gradle.properties

```diff
@@ -6,7 +6,7 @@ h2oMajorVersion=3.10.0
 # Name of H2O major version
 h2oMajorName=turing
 # H2O Build version, defined here to be overriden by -P option
-h2oBuild=7
+h2oBuild=9
```

Comment:
There are also some small additional changes because of the bump. It's good to have them separated I think